website/docs/docs/build/view-documentation.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "View documentation"
+description: "Learn how good documentation for your dbt models helps stakeholders discover and understand your datasets."
+id: "view-documentation"
+---
+
+dbt provides an intuitive and scalable way to write, version-control, and share documentation for your dbt models. You can write [descriptions](/docs/build/documentation#adding-descriptions-to-your-project) (in plain text or markdown) for each resource, and then generate and view documentation to gain shared context for your dbt project.
```

Comment:
It might be beneficial to have this opening focus more on the viewing of docs: How about something like
```suggestion
dbt provides intuitive and scalable tools for viewing your dbt documentation. Detailed documentation is essential for your developers and other stakeholders to gain shared context for your dbt project.
```